drop-database.md

```diff
@@ -1,19 +1,66 @@
 ---
 title: DROP DATABASE
-toc: true
+toc: false
 ---
 
-## Description
+The `DROP DATABASE` [statement](sql-statements.html) removes a database and all its objects from a CockroachDB cluster. 
+
+<div id="toc"></div>
 
 ## Synopsis
 
 {% include sql/diagrams/drop_database.html %}
 
-## Privileges
+## Required Privileges
+
+The user must have the `DROP` [privilege](privileges.html) on the database and all included tables. 
```

Comment:
how about: "all tables in the database" instead? just a nit, feel free to ignore.